io/eolearn/io/processing_api.py

```diff
@@ -230,3 +293,75 @@ def _extract_bands(self, eopatch, images, shape):
             bands = [np.round(band * norm, 4) for band, norm in zip(bands, norms)]
 
         eopatch[self.bands_feature] = np.asarray(bands).reshape(*shape, num_bands)
+
+    def _add_meta_info(self, eopatch):
+        eopatch.meta_info['maxcc'] = self.maxcc
+        eopatch.meta_info['time_difference'] = self.time_difference
+
+
+class SentinelHubDEMInputTask(SentinelHubInputBase):
+    ''' A processing API input task that downloads the digital elevation model
+    '''
+    def __init__(self, dem_feature, size=None, resolution=None, cache_folder=None, config=None,
+                 max_threads=None):
```

Comment:
Parameter `max_threads` is not passed forward to parent class.

Reply:
Done.